examples/chip-tool/templates/commands.zapt

```diff
@@ -60,6 +65,14 @@ static void OnInt16sAttributeResponse(void * context, int16_t value)
     command->SetCommandExitStatus(true);
 }
 
+static void OnInt64uAttributeResponse(void * context, uint64_t value)
+{
+    ChipLogProgress(chipTool, "Int64u attribute Response: %" PRIu64, value);
+
+    ModelCommand * command = reinterpret_cast<ModelCommand *>(context);
+    command->SetCommandExitStatus(true);
+}
+
```

Comment:
Strangely this block is added to the template, but I do not see it added to `examples/chip-tool/commands/clusters/Commands.h` ? Do you really need it or did you forget a `git add` ?

Reply:
It is added here: https://github.com/project-chip/connectedhomeip/pull/5059/files#diff-85c75393ac1e7ad6454d4a7c4dd520b6b3e9107b820240eef764d5ad34f30558R80-R87